common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -115,4 +115,15 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2020, 9, 4),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-subs-banner-new-year-copy-test",
+    "Test subscription banner copy variants",
+    owners = Seq(Owner.withGithub("jfsoul")),
+    safeState = On,
```

Comment:
Does safeState have to be `On`? Most of our tests have safeState as `Off` since this restores default behaviour.
This can help guide 24/7 and other teams around what to toggle off to fix issues

Reply:
No this was just for (lazy) local testing, I will swap this.